.github/workflows/main.yaml

```diff
@@ -26,8 +26,8 @@ jobs:
       run: |
         scripts/alpine-setup.sh
         make test
-  build:
-    name: Build
+  build-darwin:
```

Comment:
Would this be a good fit for using the _matrix_ functionality to avoid some repetition?
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
yes working on it